device/iot-device-samples/android-sample/app/src/main/java/com/iothub/azure/microsoft/com/androidsample/MainActivity.java

```diff
@@ -23,19 +29,12 @@ protected void onCreate(Bundle savedInstanceState) {
 
         try {
             SendMessage();
-        }
-        catch(IOException e1)
-        {
-            System.out.println("Exception while opening IoTHub connection: " + e1.toString());
-        }
-        catch(Exception e2)
-        {
+        } catch (Exception e2) {
             System.out.println("Exception while opening IoTHub connection: " + e2.toString());
         }
     }
 
-    public void SendMessage() throws URISyntaxException, IOException
-    {
+    private void SendMessage() throws URISyntaxException, IOException {
```

Comment:
As per our coding guidelines we start braces in next line. 
Refer to documentation - https://github.com/Azure/azure-iot-sdk-java/blob/master/.github/CODING_GUIDELINES.md#space-and-braces

Reply:
Out of curiosity, why aren't you following IntelliJ's and most Java developers' code conventions?

Reply:
We have gathered our coding conventions from various sources across the web coding in Java and this document shows a compilation of them. There are lot of defaults in intelliJ which do not abide by guidelines and a developer will have to change those settings to follow CG. You have encountered one such change here.